src/middleware/cors.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from "express";
+import cors from "cors";
+
+const router = Router();
+
+const corsOptions = router.use(
+  cors({
+    origin: "*",
```

Comment:
Não é bom usar *, essa config quanto mais fechada melhor. Tentar usar localhost por enquanto, depois que a gente for subir para a cloud iremos melhorar isso aqui.